packages/react/src/components/DatePicker/DatePicker-test.js

```diff
@@ -311,7 +311,9 @@ describe('Single date picker', () => {
     );
 
     userEvent.click(screen.getByText('clear'));
-    expect(screen.getByLabelText('Date Picker label')).toHaveValue('');
+    expect(screen.getByLabelText('Date Picker label')).toHaveValue(
```

Comment:
If we are testing that the value should clear when the button is pressed, this should test that clearing the selection works as intended. What has changed that is causing this test to fail now?

Reply:
@tw15egan .  Yes ! .. You are correct.  I'll check this and do the necessary changes and update it .

Reply:
@tw15egan : I have removed the changes on the DatePicket-test.js and made only this change, to get the test working.  
```
const [date, setDate] = useState('');
const [date, setDate] = useState();
```